### DISCUSSION TITLE
Add support for embed private links

### DISCUSSION THREAD

**Discussion by jalberto** [outline/outline#1468] [General] [answered]
My main use case is GDrive, as my GAccount is already in Outline, why not to use it to insert directly from Gdrive (without publishing to web).
This can be useful for other services where you want to embed private documents for your team, this embedded documents will not show up in a public shared page

**Comment by tommoor** [comment]
@jalberto the problem is I'm not aware of any way that Google makes it possible to embed private documents in another webpage. Are you aware of any services that achieve this?

**Comment by jalberto** [comment]
Notion is able to embed a preview

**Comment by tommoor** [comment, marked as answer]
Hey @jalberto – we just deployed support for embedding private Google Drive links, thanks for prompting the question.

**Reply by finsrud** [reply]
Thank you for adding this!

Notion and Slab also allow for embedding of other kinds of files stored in Google Drive (e.g. PDFs, Word). Is this something you could add to Outline?

**Reply by tommoor** [reply]
Yes, although I'd imagine the implementation would be a bit different – more of an "attachment" to a document, or are you referring to the same approach of just pasting a link?

**Reply by haxplorer** [reply]
From which version is this feature available, @tommoor ?

**Reply by tommoor** [reply]
[v0.48.1](https://github.com/outline/outline/releases/tag/v0.48.1) – complete release notes are on v0.48.0 but the docker image was not functional.

**Comment by jalberto** [comment]
Thanks!
